Libplanet/Tx/ITransaction.cs

```diff
@@ -1,15 +1,65 @@
+using System;
 using System.Collections.Immutable;
 using Bencodex.Types;
 using Libplanet.Action;
+using Libplanet.Blocks;
+using Libplanet.Crypto;
 
 namespace Libplanet.Tx
 {
     /// <summary>
-    /// Similar to <see cref="ITxExcerpt"/> except that it has <see cref="Signature"/>,
-    /// <see cref="SystemAction"/> and <see cref="CustomActions"/> as well.
+    /// An abstract interface for a transaction.  Unlike <see cref="Transaction{T}"/>, it deals
+    /// with custom actions in a non-generic way.  Instead, they are represented as
+    /// <see cref="IAction"/>.
     /// </summary>
-    public interface ITransaction : ITxExcerpt
+    public interface ITransaction
     {
+        /// <summary>
+        /// The unique identifier derived from this transaction's content including actions and
+        /// signature.
+        /// </summary>
+        /// <seealso cref="TxId"/>
+        TxId Id { get; }
+
+        /// <summary>
+        /// The number of previous <see cref="Transaction{T}"/>s committed by
+        /// the <see cref="Signer"/> of this transaction.  This nonce is used for preventing
+        /// replay attack.
+        /// </summary>
+        long Nonce { get; }
+
+        /// <summary>
+        /// A <see cref="PublicKey"/> of the account who signs this transaction.
```

Comment:
Isn't it the `Address` of the account instead of `PublicKey`?

Reply:
Amended.